docs/index.md

```diff
@@ -15,12 +15,24 @@ at [Leiden University Medical Center](https://www.lumc.nl/).
 You can run this pipeline using
 [Cromwell](http://cromwell.readthedocs.io/en/stable/):
 
+First download the latest version of the pipeline wdl file(s) and 
+from the
+[releases page](https://github.com/biowdl/PacBio-subreads-processing/releases).
+
 ```bash
 java \
     -jar cromwell-<version>.jar \
     run \
+    -o options.json \
     -i inputs.json \
-    talon-wdl.wdl
+    pacbio-subreads-processing.wdl
+```
+
+Where `options.json` contains the following json:
+```json
+{
+    "final_workflow_log_dir": "/path/to/logs/folder"
+}
```

Comment:
I think mentioning the `final_workflow_outputs_dir` and `use_relative_output_paths` would be more relevant here.